### ISSUE TITLE
DdLogs not linked to session/view after upgrade to v1.5.1

### ISSUE THREAD

**Issue by stevoland** [DataDog/dd-sdk-reactnative#462]
**Describe what happened**

After upgrading from v1.2.1 to v1.5.1 app logs no longer appear in the "Logs" tab of view in RUM. The logs still appear in the main DD "Logs" search but are missing attributes related to session and view:

Log from v1.5.1:
<img width="636" alt="Screenshot 2023-04-21 at 12 08 48" src="https://user-images.githubusercontent.com/30143/233958891-5c071c26-c441-4ddc-a5ea-249516539249.png">

Log from v1.2.1:
<img width="645" alt="Screenshot 2023-04-23 at 11 57 30" src="https://user-images.githubusercontent.com/30143/233959105-1a6e0ccd-65e9-45cb-99f9-831a45ea18b7.png">

I don't see any related config changes between these releases so I'm assuming it's a bug

**Steps to reproduce the issue:**

```ts
import { DdLogs } from '@datadog/mobile-react-native'

// Call from a react view
export const testLog = () => {
  DdLogs.info('test log', { test: 'test'})
}
```

**Describe what you expected:**

Logs should have `session_id` and `view.id` attributes and be visible in "Logs" tab of the relevant view

**Additional context**

-   Version of the SDK: 1.5.1
-   Version of React Native: 0.69.4

**Comment by louiszawadzki** [comment]
Hi @stevoland, thanks for reaching out!

This is indeed a bug on our side, I apologize for this.
The bug is present in the React Native SDK since version `1.5.0`.
We're currently working on it and will release a fix for it very soon.

I'll let you know once a fix is available.

**Comment by stevoland** [comment]
@louiszawadzki I'm also trying (for the first time) to use `DdRum.addTiming()` but am not seeing relevant attributes on views as described here https://github.com/DataDog/dd-sdk-reactnative/issues/354#issuecomment-1339668501.

Might this be related? Or should I raise another ticket? Or could it be user error?

**Comment by louiszawadzki** [comment]
Hi @stevoland, 

When you add a timing you need to specify a name for that timing, for instance `DdRum.addTiming('my_timing')`.

Once this is done, you should see the timing in the RUM View:

![image](https://user-images.githubusercontent.com/8973379/234560150-f50475a9-64fe-4a1a-bf4d-227e4dd7c45d.png)

If you don't see the timing, can you try to add a measure with the "+ Add" button on the top of the facets list and see if it appears in the facets list afterwards?

If you're not seeing anything here either, you can reach out to [our support team](https://www.datadoghq.com/support/) that will be more able to help you.

**Comment by stevoland** [comment]
@louiszawadzki Adding the measure is the step that I was missing thanks 👍 

Sorry for polluting this ticket

**Comment by stevoland** [comment]
Fixed for me in v1.5.2 - Logs are linked to view. Although I have another issue I'll raise a separate ticket for
